pom.xml

```diff
@@ -13,8 +13,9 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <java.version>1.8</java.version>
-        <java.target>1.8</java.target>
+        <!-- TODO revert -->
```

Comment:
Actually... let's leave Java11 here, and update the CHANGELOG and README to indicate that Java11 is now required to both build and run FITS.

..but please remove the "TODO" comment.

Reply:
Will do. Thanks for the catching that I left that in